backend/apps/accounts/models.py

```diff
@@ -48,3 +67,125 @@ def __str__(self) -> str:
     def is_admin(self):
         """Has access to the admin site?"""
         return self.is_staff
+
+
+class Profile(models.Model):
+    """
+    A model to represent the user's profile.
+
+    Attributes
+    ----------
+    user : User
+        an instance of the User model the profile refers to
+    first_name :
+        a first name of the user
+    last_name :
+        a last name of the user
+    date_of_birth :
+        a birth date of the user
+    gender :
+        a gender of the user
+    address :
+        an address of the user
+    phone_number :
+        a phone number of the user
+    avatar_image :
+        a profile image
+    linkedin_url :
+        url to user's LinkedIn profile
+    telegram_username :
+        telegram username of the user
+    additional_info :
+        an additional information about the user
+    created_at :
+        a date profile was created
+    updated_at :
+        a date profile was last updated
+    """
+
+    user = models.OneToOneField(
+        User,
+        on_delete=models.CASCADE,
+        related_name="user_profile",
+        db_index=True,
+    )
+    first_name = models.CharField(_("First Name"), max_length=127, db_index=True)
+    last_name = models.CharField(_("Last Name"), max_length=127, db_index=True)
+    date_of_birth = models.DateField(
+        _("Birth date"),
+        validators=[MaxValueValidator(limit_value=date.today)],
+        blank=True,
+        null=True,
+    )
+    gender = models.CharField(
+        verbose_name=_("Gender"),
+        max_length=1,
+        choices=GenderChoices.choices,
+        default=GenderChoices.OTHER,
+    )
+    address = models.CharField(
+        _("Address"),
+        null=True,
+        blank=True,
+        max_length=255,
+    )
+    phone_number = PhoneNumberField(verbose_name=_("Phone number"), unique=True)
+    avatar_image = VersatileImageField(
+        _("Avatar picture"),
+        null=True,
+        blank=True,
+        upload_to="profile_images/",
+    )
+    linkedin_url = models.URLField(
+        _("LinkedIn profile"),
+        null=True,
+        blank=True,
+        max_length=255,
+    )
+    telegram_username = models.CharField(
+        _("Telegram username"),
+        null=True,
+        blank=True,
+        max_length=32,
+        validators=[TelegramUsernameValidator()],
+    )
+    additional_info = models.TextField(
+        _("Additional Info"),
+        null=True,
+        blank=True,
+        max_length=1000,
+    )
+    created_at = models.DateTimeField(_("Date created"), auto_now_add=True)
+    updated_at = models.DateTimeField(_("Date updated"), auto_now=True)
+
+    class Meta:
+        verbose_name = _("Profile")
```

Comment:
I guess there is no need to add this as Django does it automatically. It takes model name for a singular one, and then just adds 's' to plural. If it doesn't contradict the grammar (like Ox -> Oxen), no need to specifically put it here.

Reply:
I did that in order to enable django.translation feature for these names in admin panel.

Reply:
That makes sense, sorry